service/cloudconfig/cloud_config.go

```diff
@@ -56,8 +56,9 @@ func New(config Config) (*CloudConfig, error) {
 func (c *CloudConfig) NewMasterTemplate(customObject awstpr.CustomObject, certs certificatetpr.CompactTLSAssets, keys randomkeytpr.CompactRandomKeyAssets) (string, error) {
 	var err error
 
-	// TODO remove defaulting as soon as custom objects are configured.
-	if !key.HasClusterVersion(customObject) {
+	// Default the version if it is not configured or we are using Cloud Formation.
+	// TODO Remove once Cloud Formation migration is complete.
```

Comment:
We're using the version to trigger the Cloud Formation logic. So we need to set the version here to get the correct cloud config.